### DISCUSSION TITLE
希望给一个阿里云函数计算的例子

### DISCUSSION THREAD

**Discussion by liziwl** [walinejs/waline#744] [Q&A] [answered]
[阿里云函数计算](https://fcnext.console.aliyun.com/)应该同 [百度云函数](https://waline.js.org/guide/server/baidu-cfc.html)的例子类似但是我部署的时候测试调用失败。

![image](https://user-images.githubusercontent.com/23000702/148251674-112d6805-c410-48dc-9012-d965bb3740d2.png)

不知道官方能不能给一个例子。

**Comment by liziwl** [comment]
![image](https://user-images.githubusercontent.com/23000702/148252056-dc62e926-4a15-4b27-94a7-9712e1258ae6.png)

![image](https://user-images.githubusercontent.com/23000702/148252111-15b73ec4-8936-417c-ab2d-eed735f29df2.png)

![image](https://user-images.githubusercontent.com/23000702/148252332-2b2d2eeb-15a2-43ce-a2e8-2543b6269416.png)

阿里云部署的代码不加callback会报错，我就加了一个空的。

**Comment by lizheming** [comment]
好的，这个我之后弄一下~

**Comment by lizheming** [comment, marked as answer]
https://waline.js.org/guide/server/aliyun-fc.html 已添加示例，部署方式见文档

**Reply by liziwl** [reply]
太棒了，已按照文档成功部署。
